store/tikv/2pc.go

```diff
@@ -382,11 +387,15 @@ func (c *twoPhaseCommitter) commitSingleBatch(bo *Backoffer, batch batchKeys) er
 		bo = NewBackoffer(commitPrimaryMaxBackoff, bo.ctx)
 	}
 
-	resp, err := c.store.SendKVReq(bo, req, batch.region, readTimeoutShort)
+	resp, err := c.store.SendReq(bo, req, batch.region, readTimeoutShort)
 	if err != nil {
 		return errors.Trace(err)
 	}
-	if regionErr := resp.GetRegionError(); regionErr != nil {
+	regionErr, err := resp.GetRegionError()
```

Comment:
Let's make this PR simple to contain changes for the protocol change. Any change related to refactoring could be left to later PR.